LibreNMS/Validations/WebServer.php

```diff
@@ -45,22 +45,28 @@ public function validate(Validator $validator): void
     {
         if (! app()->runningInConsole()) {
             $url = $this->removeStandardPorts(request()->url());
-            $expected = $this->removeStandardPorts(Str::finish(Config::get('base_url'), '/') . 'validate/results');
+            $base_url = Config::get('base_url');
+            $expected = $this->removeStandardPorts(Str::finish($base_url, '/') . 'validate/results');
+            $correct_base = str_replace('/validate/results', '', $url);
 
             if ($url !== $expected) {
                 preg_match($this->host_regex, $url, $actual_host_match);
                 preg_match($this->host_regex, $expected, $expected_host_match);
                 $actual_host = $actual_host_match[1] ?? '';
                 $expected_host = $expected_host_match[1] ?? "parse failure ($expected)";
-                if ($actual_host != $expected_host) {
+
+                if ($base_url == '/' && ! str_contains($actual_host, '/')) {
```

Comment:
Thoughts on updating this to:

`if ($base_url == '/' && ! str_contains($actual_host, '/') || !Str::startsWith($base_url, 'http')) {`

If I set base_url to just my.test.install as an example, the `$expected_host` match fails and it suggests that the nginx config should be updated when in fact it's base_url.

Reply:
I thought it already handled that case... I'll have to re-read things.